cpp/src/sampling/detail/graph_functions.hpp

```diff
@@ -120,6 +121,81 @@ sample_edges(raft::handle_t const& handle,
              size_t fanout,
              bool with_replacement);
 
+/**
+ * @brief Use the sampling results from hop N to populate the new frontier for hop N+1.
+ *
+ * @tparam vertex_t Type of vertex identifiers. Needs to be an integral type.
+ * @tparam label_t Type of label. Needs to be an integral type.
+ * @tparam multi_gpu Flag indicating whether template instantiation should target single-GPU (false)
+ *
+ * @param handle RAFT handle object to encapsulate resources (e.g. CUDA stream, communicator, and
+ * handles to various CUDA libraries) to run graph algorithms.
+ * @param starting_vertices The source vertices for the current hop
+ * @param starting_vertex_labels Optional labels for the source vertices for the current hop
```

Comment:
And I think these names to better be `this_frontier_vertices` and `this_frontier_vertex_labels` to be consistent with the `new_frontier...` (or better be `next_frontier...`) variable names.

Reply:
Fixed in next push